include/fiction/algorithms/physical_design/hexagonalization.hpp

```diff
@@ -299,12 +299,13 @@ template <typename CartLyt>
  * @param cartesian_layout_height Height of the Cartesian layout.
  * @param input_mode Adjust offset based on PIs relocated to the top row.
  * @param output_mode Adjust offset based on POs relocated to the bottom row.
- * @return offset.
+ * @return offset and if has to be added.
```

Comment:
Couldn't you just set the offset to 0 if it is not to be added? In my opinion, that would be the cleaner solution